components/diabloexe/baseitem.h

```diff
@@ -12,39 +12,42 @@ namespace DiabloExe
     class BaseItem
     {
     public:
-        uint32_t activTrigger;
+        uint32_t dropRate;
 
         uint8_t itemClass;
-        uint8_t equipLoc;
+        uint8_t equipType;
 
         uint16_t unknown0;
 
-        uint32_t graphicValue;
+        uint32_t invGraphicsId;
 
         uint8_t type;
-        uint8_t uniqCode;
+        uint8_t uniqueBaseItemId;
         uint16_t unknown1;
 
-        std::string itemName;
-        std::string itemSecondName;
+        std::string name;
+        std::string shortName;
         uint32_t qualityLevel;
         uint32_t durability;
         uint32_t minAttackDamage;
         uint32_t maxAttackDamage;
-        uint32_t minArmourClass; // LONG LIVE OUR GLORIOUS QUEEN
-        uint32_t maxArmourClass;
-
-        uint8_t reqStr;
-        uint8_t reqMagic;
-        uint8_t reqDex;
-        uint8_t reqVit;
-
-        uint32_t specialEffect;
-        uint32_t magicCode;
-        uint32_t spellCode;
-        uint32_t useOnce;
-        uint32_t price1;
-        uint32_t price2;
+        uint32_t minArmorClass; // LONG LIVE OUR GLORIOUS QUEEN
```

Comment:
Comment no longer maxes sense when you change the spelling to US English :p

Reply:
Fixed :)